glyphs-reader/src/font.rs

```diff
@@ -211,6 +275,43 @@ impl std::str::FromStr for NodeType {
     }
 }
 
+impl FromPlist for Component {
+    fn from_plist(plist: Plist) -> Self {
+        let Plist::Dictionary(mut dict) = plist else {
+            panic!("Component must be a dict: {:?}", plist);
+        };
+        // Glyphs v3 name has been renamed to ref; look for both
```

Comment:
I think Component::from_plist is only being used when deserializing from a v2 plist, the v3 conversion (from RawShape to Shape::Component(Component)) is done inside RawShape::try_from.

Reply:
maybe you should actually use this Component::from_plist method inside RawShape::try_from, when the latter contains a "ref"... to avoid duplicating the logic in here and over there

Reply:
Hm. I don't have the plist that made the shape that is a component anymore, but I agree it's duplicative.

Reply:
OK, if I provide `TryFrom<BTreeMap<String, Plist>> for Component` it can work for both. Sweet.